### DISCUSSION TITLE
Regarding enhancing sample app

### DISCUSSION THREAD

**Discussion by piyushjohnson** [hadiyarajesh/flower#6] [Q&A] [answered]
To showcase functionality ⭐  of database caching and network operations, may we plan onto some cool sample app.
Some points that i pondered upon
🌟  Loading multiple resources (quotes) from API page-by page 🔍 , i think this will be good starting point as cached pages(quote) will load quickly
🌟 Also further to build upon bookmark 💾 quote (locally on DB) can be included
🌟 Also with some reasonable UI 📴  updates (also considering jetpack compose if needed at some later stage)

Above mentioned points are my initial thoughts,please feel free to add more suggestions 👍  onto this

**Comment by hadiyarajesh** [comment, marked as answer]
@piyushjohnson This seems pretty cool features to showcase the sample app. Unfortunately, i won't be able to provide the time required to build these features. If you've plans, feel free to implement and create pull request. I'll merge them to main branch.
